kyaml/fn/runtime/exec/exec.go

```diff
@@ -32,5 +38,16 @@ func (c *Filter) Run(reader io.Reader, writer io.Writer) error {
 	cmd.Stdin = reader
 	cmd.Stdout = writer
 	cmd.Stderr = os.Stderr
+	if c.WorkingDir != "" {
```

Comment:
Similar to the logic for having the path validations be here, right before we run the command, should the "working directory must be set" validation also go here Instead of in `runfun.go`?

Reply:
The other entry point to the exec filter is the container runtime code, which uses the exec filter to run `docker run`. To do this validation here, we would have to set the working directory there was well.

Currently, the place in `runfn.go`, where I have the "working directory must be set" validation, is only used when running exec functions. 

I will defer to your opinion on whether the container runtime code should set the working directory for exec filters as well. The PR to do so is here: https://github.com/kubernetes-sigs/kustomize/pull/4132